lib/metro-transit.js

```diff
@@ -98,7 +96,7 @@ MetroTransit.prototype.intentHandlers = {
         response.tell(props.serviceAdvisoriesErrorSpeechOutput);
         return;
       }
-      var incidentList = _.pluck(incidents, 'Description');
```

Comment:
Whoops. I'm pretty sure I knew I needed to change this, but must have missed it in my commit.